tests/alive-tv/attrs/noreturn-2.srctgt.ll

```diff
@@ -0,0 +1,16 @@
+declare void @f() noreturn
+
+define void @src(i1 %cond) {
+  br i1 %cond, label %A, label %B
+A:
+  udiv i32 1, 0 ; cannot use unreachable due to a bug; it does not call addReturn
+  ret void
```

Comment:
I observed that this can be resolved by making Assume::toSMT() call State::addReturn(), but I guess it wouldn't be optimal for performance. What would be the best way to resolve this?